### PR TITLE
Remove public #update_shipment_total

### PR DESCRIPTION
**Pull request by mbj**
Extraction from #398

### CODE REVIEW

**Comment by mbj** [comment]
@dkubb This is ready for review.

File: core/app/models/spree/order.rb [hunk 1]
```diff
@@ -432,11 +430,6 @@ def shipping_eq_billing_address?
       (bill_address.empty? && ship_address.empty?) || bill_address.same_as?(ship_address)
     end
 
-    def set_shipments_cost
```
Comment (dkubb) [inline core/app/models/spree/order.rb]: Is this method's role being assumed by another method?

Comment (mbj) [inline core/app/models/spree/order.rb]: Due our latest refactorings that made `Spree::OrderUpdater` inconditionally performing `#update_shipment_total` from `#update` these explicit calls here where simply redundant.

File: core/spec/models/spree/order_updater_spec.rb [hunk 2]
```diff
@@ -132,15 +132,16 @@ def apply
     before do
       allow(subject).to receive_messages(
         update_payment_state:  subject,
-        update_payment_total:  undefined,
-        update_shipment_total: undefined,
+        update_payment_total:  undefined
```
Comment (dkubb) [inline core/spec/models/spree/order_updater_spec.rb]: I think there's extra whitespace here between the key and value.

**Comment by dkubb** [comment]
@mbj ok, I only have one small comment on this PR.

**Pull request merged**